src/main/java/uk/gov/hmcts/ecm/common/model/helper/Constants.java

```diff
@@ -339,21 +341,25 @@
     public static final String DUPLICATE_JURISDICTION_CODE_ERROR_MESSAGE = "Duplicate Jurisdiction Code(s) : ";
     public static final String MISSING_JURISDICTION_OUTCOME_ERROR_MESSAGE = "A Jurisdiction outcome is required before the case can be closed";
     public static final String MISSING_JURISDICTION_MESSAGE = "You cannot close this case as there is currently no jurisdiction and no jurisdiction outcome associated with it";
-    public static final String MISSING_JUDGEMENT_JURISDICTION_MESSAGE = "This case contains a Judgment which does not have a jurisdiction attached to it. To enable this case to be closed, please ensure the relevant jurisdiction(s) has been linked to the Judgment.";
```

Comment:
Did @Angela-LW not add this is yesterday? Why is it being deleted

Reply:
That was what I was asking.

Reply:
yes i added this for ECM-413, please do not delete

Reply:
restored